docs/env.mdx

```diff
@@ -27,7 +27,7 @@ Some environment variables will be added by integrations and other tooling. For
 
 ### Database Environment Variable
 
-You also need to configure a `.env` file in the `apps/database` directory. This file should be created by the setup script, so just add your `DATABASE_URL` and you should be good to go.
+You also need to configure a `.env` file in the `apps/packages/database` directory. This file should be created by the setup script, so just add your `DATABASE_URL` and you should be good to go.
```

Comment:
To correct a correction, this should be `pacakges/database` lol

Reply:
Done